src/js/data-model.jsx

```diff
@@ -103,13 +123,30 @@ var DataModel = React.createClass({
         }
 
         var dqaTable = this.props.dqa && this.props.dqa[this.props.activeTable];
-        var dqaField = dqaTable && dqaTable.fields[this.props.activeField];
-
-        if (dqaTable !== undefined && dqaField !== undefined) {
+        var dqaField = dqaTable && dqaTable[this.props.activeField];
+
+        if (dqaTable !== undefined) {
+            if (dqaField !== undefined) {
+                dqaElement =
+                    <li key='dqa'>
+                        <DQA data={dqaField} dict={this.props.dqaDict}/>
+                    </li>
+            } else {
+                // dqaField can be undefined because no DQA info is available for that field,
+                // or because we are displaying table-level info. Address the later.
+                if (!this.props.activeField) {
+                dqaElement =
```

Comment:
Same as [above](https://github.com/PEDSnet/pedsnet-catalogue/pull/3/files#r41899263). Also indent within the block.